GameBoard/Controllers/GameEventParticipationController.cs

```diff
@@ -48,6 +53,35 @@ public async Task<IActionResult> ExitGameEvent(ExitGameEventViewModel exitGameEv
             return RedirectToAction("Index", "Home");
         }
 
+        [HttpGet]
+        public async Task<IActionResult> RemoveFromGameEvent(int gameEventId, string userName)
+        {
+            var gameEvent = await _gameEventService.GetGameEventAsync(gameEventId);
+
+            if (gameEvent == null)
+            {
+                return Error.FromController(this).Error("Error!", "Game event you're trying to remove doesn't exist.", HttpStatusCode.NotFound);
+            }
+
+            return View("RemoveFromGameEvent", new RemoveFromGameEventViewModel
+                {
+                    GameEventId = gameEventId,
+                    GameEventName = gameEvent.Name,
+                    UserName = userName
+                });
```

Comment:
Use an extension method, as we do everywhere else when converting from DTO.

Reply:
Fixed.